spotbugs/src/main/java/edu/umd/cs/findbugs/ba/jsr305/TypeQualifierResolver.java

```diff
@@ -51,37 +51,30 @@ public class TypeQualifierResolver {
     static final ClassDescriptor elementTypeDescriptor = DescriptorFactory
             .createClassDescriptor(ElementType.class);
 
-    static final ClassDescriptor androidNullable = DescriptorFactory.createClassDescriptor("android/support/annotation/Nullable");
-
-    static final ClassDescriptor androidNonNull = DescriptorFactory.createClassDescriptor("android/support/annotation/NonNull");
-
-    static final ClassDescriptor androidxNullable = DescriptorFactory.createClassDescriptor("androidx/annotation/Nullable");
-
-    static final ClassDescriptor androidxNonNull = DescriptorFactory.createClassDescriptor("androidx/annotation/NonNull");
```

Comment:
This line of code still uses the code above on line 58

`private static final String ANDROIDX_ANNOTATION = "androidx/annotation/";`